packages/fxa-payments-server/public/locales/fr-FR/main.ftl

```diff
@@ -0,0 +1,6 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+document =
+  .title = Enregistrements Lockbox
```

Comment:
Why is fr-FR included but no other locales?

Even though this string will be overwritten by pontoon, it seems out of place in this patch.

Which brings up another point -- we probably need to ping the pontoon folks to add this new server.

Reply:
I included this just for testing, thought i would be easier to review. I'll remove the patch for now